notebooks/demo.ipynb

```diff
@@ -0,0 +1,850 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2e3f5e11-e67d-4c45-b47d-a5923cfd9692",
+   "metadata": {},
+   "source": [
+    "# METL-Sim in Open Science Grid\n",
+    "\n",
+    "This notebook serves allows for the deployment of the Rosetta framework Open Science Grid to run FastRelax from the Rosetta software. This notebook supports random creation of millions of variants to run the FastRelax protocol on the open science grid. The interactive functions in this notebook do not represent the full power fo the `metl-sim` repository; if you wish to customize your simulation or variants please reference the metl-sim [github](https://github.com/gitter-lab/metl-sim) readme. \n",
```

Comment:
"Open Science Grid" is a forbidden term. What is their preferred name?

Be attentive to capitalization: GitHub

Reply:
> "Open Science Grid" is a forbidden term. 

Can you please expand, this term is used in almost every paragraph throughout the entire paper.

Reply:
> Be attentive to capitalization: GitHub

I think their are many small details that are "missing" right now in the notebooks current state.